packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -26,10 +26,9 @@ export interface VariableModel extends Omit<raw.VariableModel, 'hide' | 'descrip
   datasource: DataSourceRef | null;
 }
 
-export interface Dashboard extends Omit<raw.Dashboard, 'templating' | 'annotations' | 'thresholds' | 'panels'> {
+export interface Dashboard extends Omit<raw.Dashboard, 'templating' | 'annotations' | 'panels'> {
   panels?: Array<Panel | RowPanel | raw.GraphPanel | raw.HeatmapPanel>;
   annotations?: AnnotationContainer;
-  thresholds?: ThresholdsConfig;
```

Comment:
This introduce a breaking change. The process is usually deprecate attributes before removing them.

Reply:
I don't think those are used anywhere outside of the core?

Reply:
Weird, I don't see `thresholds` property on the `raw.Dashboard`, so it looks like this overwrite wasn't used in the first place.

Reply:
It shouldn't be a breaking change, everything i removed from the veneer should be emitted from the raw generated file. I'll double-check.